example/android/javalib/3-linting/build.mill

```diff
@@ -0,0 +1,108 @@
+  // Set path to the custom `baseline.xml` file. It is usually at the root of the project
+  def androidLintBaselinePath = Task { Some(PathRef(millSourcePath / "lint-baseline.xml")) }
+
+  // Set the linting report to be generated as a text file
+  def androidLintReportFmt = Task { AndroidLintReportFormat.Txt }
+
+  // Set to true (default), stops the build if errors are found
+  def abortOnError = false
+
+  // Specify additional flag options for Android Lint not defined in `AndroidLintModule`
+  def androidLintArgs: T[Seq[String]] = T {
+
+    // Set path to generated `.jar` files in this case (or `.class` files)
+    val cp = runClasspath().map(_.path).filter(os.exists).mkString(":")
+
+    // Set path to the location of the project source codes
+    val src = sources().map(_.path).filter(os.exists).mkString(":")
+    Seq(
+      "--classpath",
+      cp,
+      "--sources",
+      src
+    )
+  }
+
+}
+
+/** See Also: app/AndroidManifest.xml */
+
+// The `AndroidManifest.xml` file shown above is flawed with hardcoded string linting problem so it can be spoted by running the `androidLintRun` task.
+
+/** See Also: app/resources/values/colors.xml */
+/** See Also: app/resources/values/strings.xml */
+/** See Also: app/src/main/java/com/helloworld/app/MainActivity.java */
+/** See Also: app/lint.xml */
+
+// The `lint.xml` file shown above ignores the `MissingApplicationIcon` error in the code as
+// the project is for demostration. There are no icons for the demo project.
+
+////SNIPPET:END
+
+/** Usage
+
+> ./mill show app.androidLintRun  # Display full path to the linting report in text file
+".../out/app/androidLintRun.dest/report.txt"
+
+> ls app/ # List out all files in the app directory to check lint-baseline file exists
+AndroidManifest.xml
+lint-baseline.xml
+lint.xml
+resources
+src
+
+> cat out/app/androidLintRun.dest/report.txt # Display content of the linting report
+AndroidManifest.xml:3: Error: Avoid hardcoding the debug mode; leaving it out allows debug and release builds to automatically assign one [HardcodedDebugMode]
+
+> sed -i.bak 's/ android:debuggable="true"//g' app/AndroidManifest.xml # Fix the HardcodedDebugMode warning issue from `AndroidManifest.xml`
+
+> ./mill app.androidLintRun # Rerun it for new changes to reflect
+
+> cat out/app/androidLintRun.dest/report.txt # Check the content of report again
+lint-baseline.xml: Information: 1 errors/warnings were listed in the baseline file (lint-baseline.xml) but not found in the project; perhaps they have been fixed? Unmatched issue types: HardcodedDebugMode [LintBaseline]
+
+*/
+
+// == Linting with Custom Configurations
+
+// The commands below set the serverity level of `MissingApplicationIcon` to `warning` in the `lint.xml` file that was shown above
+// and run the `androidLintRun` task. Some output of the changes is shown below.
+
+/** Usage
+
+> sed -i.bak 's/severity="ignore"/severity="warning"/g' app/lint.xml # Set `MissingApplicationIcon` severity level to warning
```

Comment:
I think this example is also missing the usage of the `SuppressLint` annotation in the code (see https://developer.android.com/studio/write/lint#configure-lint-checking-in-java-or-kotlin), so it is not clear if it is working or not.

Same for suppression in XML https://developer.android.com/studio/write/lint#configure-lint-checking-in-xml